src/utils/graph/common.js

```diff
@@ -1,6 +1,8 @@
 /**
  * PI / 2 constant
  */
+
+/* eslint-disable id-length */
```

Comment:
If this is applied to the entire file I recommend putting this at the top / first line of this file for clarity.